modules/olmv1-catalog-selection-by-name.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+// * extensions/catalogs/olmv1-catalog-content-resolution.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-catalog-selection-by-name_{context}"]
+= Catalog selection by name
+
+
+When a catalog is added to a cluster, a label is created by using the value of the `metadata.name` field of the catalog custom resource (CR). In the CR of an extension, you can specify the catalog name by using the `spec.source.catalog.selector.matchLabels` field. The value of the `matchLabels` field uses the following format:
+
+.Example value
+[source,text]
+----
+olm.operatorframework.io/metadata.name: <catalog_metadata_name>
+----
+
+The following example resolves the `<example_extension>-operator` from the catalog with the `openshift-redhat-operators` label:
```

Comment:
```suggestion
The following example resolves the `<example_extension>-operator` package from the catalog with the `openshift-redhat-operators` label:
```

Reply:
Also, should this be _a catalog_ to suggest the package could come from more than one catalog at any given time (assuming I understand what is going on here)?
```suggestion
The following example resolves the `<example_extension>-operator` from a catalog with the `openshift-redhat-operators` label:
```